src/app/components/AddressBox/index.tsx

```diff
@@ -22,14 +22,7 @@ export const AddressBox = memo((props: Props) => {
   }
 
   return (
-    <Box
-      direction="row"
-      align="center"
-      round="5px"
-      border={{ color: 'brand' }}
-      pad={{ right: 'small' }}
-      width="fit-content"
-    >
+    <Box direction="row" align="center" round="5px" pad={{ right: 'small' }} width="fit-content">
```

Comment:
Don't know if this change was intentional for validator details too. I think the validator address looks awkward after:

| Before | After |
| --- | --- |
|  ![wallet oasisprotocol org_account_oasis1qzm5madmsvecf4k4sp5qmpj8d3zw7tuuuslx9pxx_stake](https://user-images.githubusercontent.com/3758846/180472692-61566cc2-a946-4c07-90b7-47a61b84bcbe.png) |  ![localhost_3000_account_oasis1qzm5madmsvecf4k4sp5qmpj8d3zw7tuuuslx9pxx_stake (1)](https://user-images.githubusercontent.com/3758846/180472686-1e2ced43-2611-4bd6-8aec-a3af863740af.png) |

Reply:
you mean without border? it was intentional, because with border it was looking like na input that user can interact with (at least that's what I was always thinking).